tb_plugin/torch_tb_profiler/profiler/gpu_metrics_parser.py

```diff
@@ -151,6 +155,27 @@ def calculate_occupancy(self, steps_start_time, steps_end_time):
             if total_time > 0:
                 self.avg_occupancy_per_device[gpu_id] = total_occupancy / total_time
 
+    def calculate_tc_ratio(self, steps_start_time, steps_end_time):
```

Comment:
could we move all the logic to calculate tensor core ratio to a consolidated place instead of spread across multiple files?